create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -36,50 +44,52 @@ include::modules/op-resolver-hub-config.adoc[leveloffset=+2]
-
-You can use the cluster resolver to specify a pipeline or task that is defined in a namespace on the {OCP} cluster where {pipelines-title} is running.
-
-In particular, you can use the cluster resolver to access tasks that {pipelines-shortname} provides in its installation namespace, which is normally the `openshift-pipelines` namespace.
-
-include::modules/op-resolver-cluster-config.adoc[leveloffset=+2]
-include::modules/op-resolver-cluster.adoc[leveloffset=+2]
-
-include::modules/op-resolver-cluster-tasks-ref.adoc[leveloffset=+1]
-
 [id="resolver-git-anon_{context}"]
-== Specifying a remote pipeline or task with anonymous Git cloning
+== Specifying a remote pipeline, task, or step action with anonymous Git cloning
 
-You can use the Git resolver to access a remote pipeline or task from a Git repository. The repository must include a YAML file that defines the pipeline or task. For anonymous access, you can clone repositories with the resolver without needing authentication credentials.
+You can use the Git resolver to access a remote pipeline, task, or `StepAction` definition from a Git repository. The repository must include a YAML file that defines the pipeline or task. For anonymous access, you can clone repositories with the resolver without needing authentication credentials.
 
 include::modules/op-resolver-git-config-anon.adoc[leveloffset=+2]
 include::modules/op-resolver-git-anon.adoc[leveloffset=+2]
 
 [id="resolver-git-scm_{context}"]
-== Specifying a remote pipeline or task with an authenticated API
+== Specifying a remote pipeline, task, or step action with an authenticated Git API
 
-You can specify a remote pipeline or task from a Git repository by using the Git resolver. The repository must contain a YAML file that defines the pipeline or task. You can securely access repositories by using an authenticated API, which supports user authentication.
+You can specify a remote pipeline, task, or `StepAction` definition from a Git repository by using the Git resolver. The repository must contain a YAML file that defines the pipeline or task. You can securely access repositories by using an authenticated API, which supports user authentication.
 
 include::modules/op-resolver-git-config-scm.adoc[leveloffset=+2]
 include::modules/op-resolver-git-scm.adoc[leveloffset=+2]
 include::modules/op-resolver-git-override-scm.adoc[leveloffset=+2]
 
 [id="resolver-http_{context}"]
-== Specifying a remote pipeline or task by using the HTTP resolver
+== Specifying a remote pipeline, task, or step action by using the HTTP resolver
 
-You can specify a remote pipeline or task from an HTTP or HTTPS URL by using the HTTP resolver. The URL must point to a YAML file that defines the pipeline or task.
+You can specify a remote pipeline, task, or `StepAction` definition from an HTTP or HTTPS URL by using the HTTP resolver. The URL must point to a YAML file that defines the pipeline, task, or step action.
```

Comment:
```suggestion
You can specify a remote pipeline, task, or `StepAction` definition from an HTTP or HTTPS URL by using the HTTP resolver. The URL must point to a YAML file that defines the pipeline, task, or StepAction.
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".

I use "StepAction definition" earlier, but "define a definition" would be clearly unsuitable.